www/pages/permissions.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Permissions and project access
+permalink: /permissions
+---
+
+There are several levels of permissions to control access to your projects in Mergin.
+
+## Public and private project
+
+A private project is a project that only you and those who you choose to share the project with will have access to the files, history and settings. A public project is a project that everyone (including those who are not registered with Mergin) can view the data and project history. They cannot contribute to your public project unless you have shared the project with them with write access (see below for permissions)
+
+## Project permissions
+
+There are three levels of permissions when you [share a project](sharing-project) with other Mergin users:
+
+- **Read**: users will be able to view the project data and [history](project-details)
+- **Write**: in addition to the **Read** access, users can edit data (adding/removing/updating files and features). This is the most appropriate permission level you should assign to those who you want to contribute to your projects
+- **Owner**: in addition to the **Write** access, users with this level of access can delete the project or transfer it to another user or organisation
+
+## Organisation permissions
+
+Similar to project permissions, you can assign permissions to the members of your organisation. By default, all members will be assigned **Read** access. You can change their global permissions under [members' settings](working-with-organisation) or the project level (see above section).
+
+For members in an organisation, you have 4 levels of permissions:
+
+- - **Read**: users will be able to view the projects data and [history](project-details)
+- **Write**: in addition to the **Read** access, users can edit data (adding/removing/updating files and features).
+- **Admin**: in addition to the **Write** access, users with this level of access can delete the project or transfer it to another user or organisation

```

Comment:
And maybe more importantly, they can create new projects, and add/remove org members